index.js

```diff
@@ -454,7 +472,37 @@ function translateDataType(schema, isForeignReference = false) {
 		propertyType = "number";
 	} else if (schema.type === "array") {
 		if (!schema.items.type) {
-			propertyType = `Array<${translateDataType(schema.items, isForeignReference)}>`;
+			// To support when a schema has an array type consisting of references to oneOf $refs.
+			// See PolicyResponse -> AdaJsonPolicy
+			/*
+			e.g..
+			Transform 
```

Comment:
This block comment can be removed if desired. This was to help make sense of this snippet.

Reply:
- [ ] From what I remember this block doesn't need to be added. Translate data type already figures out what to do with oneOf.

Reply:
Tis is true.

Reply:
Removed